src/less/daterange-dropdown.less

```diff
@@ -0,0 +1,38 @@
+.daterange__dropdown .menu__divider {
```

Comment:
Hey team -- sorry, i don't think I was clear here and also did not realize the need for nesting (which falls into the good case I noted in my original comment)

These styles are still nested, even if we have removed the brackets and indentation. The classes will only be applied based on the hierarchy we've noted here. 

While we generally want to avoid that, in this case as I said it makes sense because these are modifying existing classes to fill a customization need. 

So this is my main takeaway here:

1. We should revert back this change, because this is good nesting, and the bracket syntax is easier to follow.
2. This should not be its own stylesheet, it should live inside the dropdowns.less file.

If you see that file you'll notice it has some nested blocks to handle the various types of dropdowns that we have. This is a new type of dropdown so it should fall inside of that existing structure.

Let me know if that makes sense or if y'all have any questions.

Sorry again for the confusion. I hope the intent is clearer this time around.

Reply:
done